taxi/views.py

```diff
@@ -28,6 +30,20 @@
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def assign_driver_to_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    car.drivers.add(request.user)
+    return redirect(reverse("taxi:car-detail", args=[pk]))
+
+
+@login_required
+def unassign_driver_from_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    car.drivers.remove(request.user)
```

Comment:
Ensure that `request.user` is a `Driver` instance before removing it from `car.drivers`. This prevents potential errors if a non-driver user attempts to unassign themselves from a car.